app/models/qa/linked_data/config/context_property_map.rb

```diff
@@ -44,12 +46,16 @@ def drillable?
           @drillable
         end
 
+        def group?
+          group_id.present?
+        end
+
         private
 
           def extract_label
             i18n_key = Qa::LinkedData::Config::Helper.fetch(property_map, :property_label_i18n, nil)
             default = Qa::LinkedData::Config::Helper.fetch(property_map, :property_label_default, nil)
-            return I18n.t(i18n_key, default) if i18n_key.present?
+            return I18n.t(i18n_key, default: default) if i18n_key.present?
```

Comment:
This was a bug.  It wasn't caught because the actual fetching of the I18n is stubbed in the test.